packages/wrangler/src/miniflare-cli/assets.ts

```diff
@@ -335,6 +335,11 @@ async function generateAssetsFetch(
 
 	const generateResponse = (request: MiniflareRequest) => {
 		const url = new URL(request.url);
+		let assetName = url.pathname;
+		try {
+			//it's possible for someone to send a URL like http://fakehost/abc%2 which would fail to decode
+			assetName = decodeURIComponent(url.pathname);
```

Comment:
What happens in prod in this case? Do we just silently ignore like we're doing here?

Reply:
Yes, this code is the same as prod's